README.md

```diff
@@ -215,16 +215,14 @@ library my_library;
 
 A file `categories.json` will be generated at the top level of the documentation tree with
 information about categories collected from objects in the source tree.  The directives
-`@category`, `@subCategory`, and `@image` are understood and saved into this json.
-Future versions of dartdoc may make direct use of the image tags.
+`@category`, and `@subCategory` are understood and saved into this json.
 
 As an example, if we document the class Icon in flutter using the following:
 
 ```dart
 /// {@category Basics}
-/// {@category Assets, Images, and Icons}
+/// {@category Assets, and Icons}
```

Comment:
I think this line can remain unchanged. Or remove it in line 238 as well... In any case I think we have an ugly 'free-form' category parsing here: This is one category called "Assets, Images, and Icons." IMO we should force the quotes, as doc directive arguments are separated by whitespace, in other directives.

Reply:
>  IMO we should force the quotes, as doc directive arguments are separated by whitespace, in other directives.

What do you mean by this? like `/// {@category "Assets and Icons"}`?

Reply:
Right. Like shell arguments, so that `{@image}` takes one parameter, and "Assets and Icons" is that one argument.